Filelist

```diff
@@ -761,29 +762,28 @@ RT_ALL =	\
 		runtime/pack/dist/opt/editorconfig/.editorconfig \
 		runtime/pack/dist/opt/editorconfig/CONTRIBUTORS \
 		runtime/pack/dist/opt/editorconfig/LICENSE* \
-		runtime/pack/dist/opt/editorconfig/mkzip.sh \
 		runtime/pack/dist/opt/editorconfig/README.md \
 		runtime/pack/dist/opt/editorconfig/autoload/*.vim \
 		runtime/pack/dist/opt/editorconfig/autoload/editorconfig_core/*.vim \
-		runtime/pack/dist/opt/editorconfig/doc/tags \
 		runtime/pack/dist/opt/editorconfig/doc/editorconfig.txt \
 		runtime/pack/dist/opt/editorconfig/ftdetect/editorconfig.vim \
 		runtime/pack/dist/opt/editorconfig/plugin/editorconfig.vim \
 		runtime/pack/dist/opt/justify/plugin/justify.vim \
 		runtime/pack/dist/opt/matchit/plugin/matchit.vim \
 		runtime/pack/dist/opt/matchit/doc/matchit.txt \
-		runtime/pack/dist/opt/matchit/doc/tags \
```

Comment:
don't remove those tags, otherwise the help won't work

Reply:
This file is in the "RT_ALL_BIN" variable, where "runtime/doc/tags" is located.
I assumed that if "doc/tags" is originally defined in this variable, then it is logical that others should be defined in the same variable.
Moreover, the variable "RT_ALL_BIN" is added to all runtime archives.